src/components/Widgets/Widgets.tsx

```diff
@@ -10,30 +9,19 @@ import type { WidgetSubvariant } from '@lifi/widget';
 import { usePathname } from 'next/navigation';
 import { useCallback, useLayoutEffect, useMemo, useState } from 'react';
 import { ThemesMap } from 'src/const/themesMap';
-import { TrackingAction, TrackingCategory } from 'src/const/trackingKeys';
-import { useUserTracking } from 'src/hooks/userTracking';
 import type { ThemeModesSupported } from 'src/types/settings';
-import { EventTrackingTool } from 'src/types/userTracking';
-import { Widget } from './Widget';
 import { WidgetEvents } from './WidgetEvents';
-import { WidgetContainer } from './Widgets.style';
 
 interface WidgetsProps {
   widgetVariant: StarterVariantType;
   activeTheme: ThemeModesSupported | undefined;
   closedWelcomeScreen: boolean;
 }
 
-export function Widgets({
-  widgetVariant,
-  activeTheme,
-  closedWelcomeScreen,
-}: WidgetsProps) {
+export function Widgets({ widgetVariant, closedWelcomeScreen }: WidgetsProps) {
   const { activeTab, setActiveTab } = useActiveTabStore();
-  const { welcomeScreenClosed, setWelcomeScreenClosed } =
-    useWelcomeScreen(closedWelcomeScreen);
+  const { setWelcomeScreenClosed } = useWelcomeScreen(closedWelcomeScreen);
   const pathname = usePathname();
-  const { trackEvent } = useUserTracking();
   const [starterVariantUsed, setStarterVariantUsed] = useState(false);
   const [_starterVariant, setStarterVariant] = useState<
```

Comment:
do we still use this?

Reply:
might be another ticket to adapt this logic elsewhere for the tabs

Reply:
A little bit for now, removing a part of it on each of my pr :s